make fails

Ubuntu 20.04
Vivado 2022.2

Vivado environment variable set OK

```
(base) abraxas3d@Jacks:~/documentation-friday/pluto_msk/firmware$ which vivado
/tools/Xilinx/Vivado/2022.2/bin/vivado
(base) abraxas3d@Jacks:~/documentation-friday/pluto_msk/firmware$
```

make fails


```
DTC     arch/arm/boot/dts/zynq-pluto-sdr-revc.dtb
make[1]: Leaving directory '/home/abraxas3d/documentation-friday/pluto_msk/firmware/linux'
dtc -q -@ -I dtb -O dts linux/arch/arm/boot/dts/zynq-pluto-sdr-revc.dtb | sed 's/axi {/amba {/g' | dtc -q -@ -I dts -O dtb -o build/zynq-pluto-sdr-revc.dtb
rm -Rf build/sdk
unzip -o build/system_top.xsa system_top.bit -d build
unzip:  cannot find or open build/system_top.xsa, build/system_top.xsa.zip or build/system_top.xsa.ZIP.
make: *** [Makefile:185: build/system_top.bit] Error 9
rm linux/arch/arm/boot/dts/zynq-pluto-sdr-revc.dtb linux/arch/arm/boot/dts/zynq-pluto-sdr.dtb linux/arch/arm/boot/dts/zynq-pluto-sdr-revb.dtb
```

Deleted repository, attempted again, same error. I attempted it again because the install above required a number of dependencies to be installed along the way, and I wanted to make sure that those installs had not caused some sort of problem. 

Here's the error message:

```
make[1]: Leaving directory '/home/abraxas3d/documentation-vancouver/pluto_msk/firmware/linux'
dtc -q -@ -I dtb -O dts linux/arch/arm/boot/dts/zynq-pluto-sdr-revc.dtb | sed 's/axi {/amba {/g' | dtc -q -@ -I dts -O dtb -o build/zynq-pluto-sdr-revc.dtb
rm -Rf build/sdk
unzip -o build/system_top.xsa system_top.bit -d build
unzip:  cannot find or open build/system_top.xsa, build/system_top.xsa.zip or build/system_top.xsa.ZIP.
make: *** [Makefile:185: build/system_top.bit] Error 9
rm linux/arch/arm/boot/dts/zynq-pluto-sdr-revc.dtb linux/arch/arm/boot/dts/zynq-pluto-sdr.dtb linux/arch/arm/boot/dts/zynq-pluto-sdr-revb.dtb

real	47m18.688s
user	99m25.825s
sys	11m14.176s

```

--- a/README.md
+++ b/README.md
@@ -13,12 +13,17 @@ The following ORI library components are used as submodules to this repository:
 5. [prbs](https://github.com/OpenResearchInstitute/prbs)
 
 ## Building
-In oder to build fpga bitstream, you need to install first Vivado. Recommended version is 2022.2. In this documentation vivado installtion path is assumed to be /opt/Xilinx/Vivado. If it is on an other directory, change the path (for example /tools/Xilinx/Vivado on keroppi) 
+In oder to build the FPGA bitstream, you first need to install Vivado. Recommended version is 2022.2. In this documentation and in the `Makefile`, the Vivado installation path is assumed to be `/opt/Xilinx/Vivado`. If it is in another directory, change the path in the steps below (for example, to `/tools/Xilinx/Vivado`) or create a symbolic link in `/opt` pointing to the actual installation directory. For example,
+```
+sudo ln -s /tools/Xilinx /opt/Xilinx
+```
+Such a symbolic link is already in place on chococat and keroppi.
 ### First, clone this repo with all submodules
 git clone --recursive https://github.com/OpenResearchInstitute/pluto_msk
 ### building bitstream only
 1. cd pluto_msk/projects/pluto/
 2. source /opt/Xilinx/Vivado/2022.2/settings64.sh
+<br>(See above about the Vivado installation path.)
 3. make
 
 For reference, on the chococat VM, the main branch takes:
@@ -56,7 +61,8 @@ sys	9m16.466s
 ```
 
 ### complete firmware
-1. Check if your vivado path is correct on this line https://github.com/OpenResearchInstitute/pluto_msk/blob/871bd130de0e6d462138bc6d2981b1c65897a0ca/firmware/Makefile#L19 
+1. Check if your Vivado path is correct on this line https://github.com/OpenResearchInstitute/pluto_msk/blob/871bd130de0e6d462138bc6d2981b1c65897a0ca/firmware/Makefile#L19 
+<br>See above about the Vivado installation path. If `/opt/Xilinx` does not contain the Vivado software, create the symbolic link described above. An alternative is to edit this line of `Makefile` every time you clone this repository.
 2. cd pluto_msk/firmware
 3. make
 

